Dockerfile_builder

```diff
@@ -1,11 +0,0 @@
-FROM golang:1.10.1-alpine3.7
```

Comment:
As far as I could tell, this file was unused

Reply:
Oh yes, this is pretty old. I don't know how we didn't even notice it still was around.